compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2759,8 +2759,7 @@ TR::Register *OMR::Power::TreeEvaluator::inlineVectorBitSelectOp(TR::Node *node,
 
    TR::Register *resReg;
 
-   if (op != TR::InstOpCode::xvmaddadp &&
-       op != TR::InstOpCode::xvmsubadp &&
+   if (op != TR::InstOpCode::xvmsubadp &&
```

Comment:
Can you please clarify what the reason behind this change is? It seems unrelated to the rest of this change AFAICT.

Reply:
Ah, I think that's a leftover from an earlier draft that I forgot to clean up. Thanks for catching that!